website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/index.mdx

```diff
@@ -31,33 +30,34 @@ Since administrators of these services within your organization are not
 necessarily the same person, this documentation refers to a **ServiceNow Admin**
 and a **Terraform Admin**.
 
-First, the Terraform Admin will configure your HCP Terraform organization with
-a dedicated team for the ServiceNow integration, and obtain a team API token for
+First, the Terraform Admin will configure your HCP Terraform organization with a
+dedicated team for the ServiceNow integration, and obtain a team API token for
 that team. The Terraform Admin will provide your organization name, the team API
```

Comment:
Can we make this a list? It's a pretty dense list as is and splitting it out would make it more readable:
```suggestion
that team. The Terraform Admin provides the following to your ServiceNow admin:
* An Organization name
* A team API token
* The hostname of your HCP Terraform instance
* Any available no-code modules or version control repositories containing Terraform configurations 
* Other required variables
```